Assets/LeapMotion/Modules/InteractionEngine/Scripts/UI/InteractionButton.cs

```diff
@@ -21,8 +24,25 @@ namespace Leap.Unity.Interaction {
   ///</summary>
   public class InteractionButton : InteractionBehaviour {
 
+    [Header("UI Control")]
+    [Tooltip("When set to false, this UI control will not be functional. Use this instead "
+           + "of disabling the component itself when you want to disable the user's "
+           + "ability to affect this UI control.")]
+    public bool controlEnabled = true;
+    public void SetControlEnabled(bool enableControl) {
```

Comment:
Any reason to have a public variable _and_ a public method that can set/reset it?